src/sass/common/_layout.scss

```diff
@@ -27,6 +27,8 @@ html {
 .Root-loader {
     width: 100vw;
     height: 100vh;
+    position: fixed;
```

Comment:
the placement of `.Root-loader` DOM element used to be in `Root.jsx`, now it is further down in `Search.jsx`, so just fix position this and good to go